storm-core/src/jvm/backtype/storm/scheduler/TopologyDetails.java

```diff
@@ -403,13 +415,23 @@ public void addResourcesForExec(ExecutorDetails exec, Map<String, Double> resour
      * Add default resource requirements for a executor
      */
     public void addDefaultResforExec(ExecutorDetails exec) {
+        Double topologyComponentCpuPcorePercent = Utils.getDouble(topologyConf.get(Config.TOPOLOGY_COMPONENT_CPU_PCORE_PERCENT), null);
```

Comment:
its just getting the set value for the config TOPOLOGY_COMPONENT_CPU_PCORE_PERCENT